README.md

```diff
@@ -34,6 +34,7 @@ The default mappings are as follows:
 - <kbd>K</kbd>, <kbd>J</kbd>: select prev or next tab
 - <kbd>r</kbd>: reload current tab
 - <kbd>R</kbd>: reload current tab without cache
+- <kbd>D</kbd>: duplicate current tab
```

Comment:
I think <kbd>SHift</kbd>+<kbd>d</kbd> is confusing, because it's not deliting. In #124, I propose <kbd>z</kbd><kbd>p</kbd> to pin/unpin a tab.  How about <kbd>z</kbd><kbd>d</kbd>?

Reply:
OK, I will fix it.